main.tf

```diff
@@ -20,32 +20,13 @@ moved {
   from = aws_s3_bucket_public_access_block.default
   to   = module.aws_s3_bucket.aws_s3_bucket_public_access_block.default
 }
-*/
+moved {
```

Comment:
It would be nice to move these moved blocks to a separate file. Perhaps a moved.tf?